api/src/pcapi/routes/backoffice_v3/templates/compliance/domain_names.html

```diff
@@ -0,0 +1,125 @@
+{% from "components/badges.html" import build_venue_badges %}
+{% from "components/forms.html" import build_form_fields_group with context %}
+{% from "components/turbo/lazy_modal.html" import build_lazy_modal with context %}
+{% extends "layouts/connected.html" %}
+
+{% block page %}
+
+    <div class="pt-3 px-5">
+        <h2 class="fw-light">Règles de conformité > Noms de domaines</h2>
+        <div>
+            <button class="btn btn-outline-primary lead fw-bold my-3"
+                    data-bs-toggle="modal"
+                    data-bs-target="#blacklist-domain-names"
+                    type="button">
+                Ajouter un ou plusieurs noms de domaines
+            </button>
+
+            <div class="modal fade"
+                 id="blacklist-domain-names"
+                 tabindex="-1"
+                 aria-labelledby="blacklist-domain-names-label"
+                 aria-hidden="true">
+                <div class="modal-dialog modal-dialog-centered">
+                    <div class="modal-content">
+                        <form action="{{ url_for("backoffice_v3_web.compliance.prepare_blacklist_domain_names") }}"
+                              name="{{ url_for("backoffice_v3_web.compliance.prepare_blacklist_domain_names") | action_to_name }}"
+                              method="get"
+                              data-turbo="true"
+                              >
+                              <div class="modal-header"
+                                   id="blacklist-domain-names-header">
+                                  <h5 class="modal-title">Bannir des noms de domaines</h5>
+                              </div>
+                              <div class="modal-body row">{{ build_form_fields_group(form) }}</div>
+                              <div class="modal-footer">
+                                  <button type="button"
+                                          class="btn btn-outline-primary"
+                                          data-bs-dismiss="modal">Annuler</button>
+                                  <button type="submit"
+                                          class="btn btn-primary">Enregistrer</button>
+                              </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+
+        <table class="table mb-4">
+            <thead>
+                <tr>
+                    <th scope="col"></th>
+                    <th scope="col">Nom de domaine</th>
+                    <th scope="col">Date</th>
+                    <th scope="col">Auteur de l'action</th>
+                    <th scope="col">Impact</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for event in history %}
+                    {% if event.extraData %}
+                        {% set domains = event.extraData.get("domains", []) | join(',') %}
```

Comment:
```suggestion
                        {% set domains = event.extraData.get("domains", []) | join(', ') %}
```
Plus lisible en espaçant pour le rendu sur la page